modules/Dasboard/components/Navbar/Navbar.tsx

```diff
@@ -1,16 +1,28 @@
 import React from "react";
 
-import { ActionIcon, Anchor, Button, Group, Menu, Title } from "@mantine/core";
+import { useRouter } from "next/router";
+
+import { Anchor, Button, Group, Menu, Title } from "@mantine/core";
 import { FaPlus } from "react-icons/fa6";
-import { useSelector } from "react-redux";
 
+import { useAppDispatch, useAppSelector } from "@/shared/redux/hooks";
+import { clearUser } from "@/shared/redux/reducers/user.reducer";
+import { EUserRole } from "@/shared/redux/rtk-apis/auth/auth.types";
 import { TRootState } from "@/shared/redux/store";
 
 import { useStyles } from "./Navbar.styles";
 
 const Navbar = () => {
-  const user = useSelector((state: TRootState) => state.auth.user);
+  const user = useAppSelector((state: TRootState) => state.authenticatedUser);
```

Comment:
Left a comment in the previous PR for the selector implementation. It should be imported from the user reducer